### PR TITLE
LPS-76154 Adapt soy to deal with predefined value and immutable value type

### PR DESCRIPTION
**Pull request by pedroqueiroz94**

### CODE REVIEW

**Comment by pedroqueiroz94** [comment]
ci:test

**Comment by pedroqueiroz94** [comment]
ci:retest

File: dynamic-data-mapping-type-select/src/main/resources/META-INF/resources/select.soy [hunk 1]
```diff
@@ -12,9 +12,10 @@
  */
 {template .render}
 	{@param name: string}
+	{@param predefinedValue: list<string>}
```
Comment (leoadb) [inline dynamic-data-mapping-type-select/src/main/resources/META-INF/resources/select.soy]: @pedroqueiroz94 Please fix the parameter order. Thanks!

Comment (pedroqueiroz94) [reply dynamic-data-mapping-type-select/src/main/resources/META-INF/resources/select.soy]: @leoadb , done!

**Pull request merged**